### ISSUE TITLE
Add banner advertising latest blog post

### ISSUE THREAD

**Issue by dcherian** [xarray-contrib/xarray.dev#223]
Could be nice to have this on the landing page.

Thoughts?

**Comment by TomNicholas** [comment]
@andersy005 we have two new blog posts to show off now - how do I change the banner to point to them?

**Comment by andersy005** [comment]
@TomNicholas, the following should work


```diff
diff --git a/src/components/layout.js b/src/components/layout.js
index 30bc242..2a95e15 100644
--- a/src/components/layout.js
+++ b/src/components/layout.js
@@ -13,11 +13,17 @@ export const Layout = ({
   url = 'https://xarray.dev',
   enableBanner = false,
 }) => {
-  const bannerTitle = ''
+  const bannerTitle = 'New blog posts are available. Check them out!'
   const bannerDescription = ''
   const bannerChildren = (
     <Text fontWeight='medium'>
-      <Link href='/href-placeholder'>Text for a future announcement</Link>
+      <Link href='/blog/xarray-kvikio'>
+        Enabling GPU-native analytics with Xarray and kvikIO
+      </Link>{' '}
+      and {''}
+      <Link href='/blog/introducing-pint-xarray'>
+        Unit-aware arithmetic in Xarray, via pint
+      </Link>
     </Text>
   )
   return (
diff --git a/src/pages/index.js b/src/pages/index.js
index b6a65a9..bfe7900 100644
--- a/src/pages/index.js
+++ b/src/pages/index.js
@@ -16,6 +16,7 @@ export default function IndexPage() {
       card={
         'https://raw.githubusercontent.com/xarray-contrib/xarray.dev/main/public/dataset-diagram-logo.png'
       }
+      enableBanner
     >
       <HeroBanner />
       <Features />
```

ping me if you run into any issues...

**Comment by TomNicholas** [comment]
Closed via #226 and #324
